src/components/Tabs/Tabs.tsx

```diff
@@ -42,17 +42,19 @@ export const Tabs = ({
       sx={containerStyles}
     >
       {data.map((el, index) => {
+        const keys = a11yProps(ariaLabel, index);
         const tab = (
           <Tab
-            key={`${el.label ?? 'tab-key'}-${index}`}
+            className={`${ariaLabel} ${value === index ? 'active' : 'inactive'}`}
```

Comment:
Not sure what is the goal of this

Reply:
It was when we had all the inactive tabs sitting in the DOM to be hidden.